ts/testUtils.ts

```diff
@@ -28,32 +29,26 @@ const printOutput = (output: Output) => {
   `);
 };
 
-export const assertSuccess = (output: Output): Output => {
-  try {
-    assertEquals(output.exitCode, 0);
-    return output;
-  } catch (e) {
-    printOutput(output);
-    throw e;
-  }
-};
+export const assertSuccess = (output: Output): Output =>
+  assertOnOutput(output, () => assertEquals(output.exitCode, 0));
 
-export const assertStdout = (output: Output, expected: string) => {
-  try {
-    assertEquals(output.stdout, expected);
-  } catch (e) {
-    printOutput(output);
-    throw e;
-  }
-};
+export const assertStdout = (output: Output, expected: string): Output =>
+  assertOnOutput(output, () => assertEquals(output.stdout, expected));
 
-export const assertStderr = (output: Output, expected: string) => {
+export const assertStderr = (output: Output, expected: string): Output =>
+  assertOnOutput(output, () => assertEquals(output.stderr, expected));
+
+export const assertOnOutput = (
```

Comment:
I found this function name a bit confusing, first because it doesn't do any assertions itself, and second because I thought it had something to do with stdout output vs our `Output` type. Maybe something like this?
```suggestion
export const printOutputOnFailure = (
```

Reply:
Done in f7b2fe22.